src/index.js

```diff
@@ -163,12 +162,12 @@ export class SlideDeck extends React.Component {
 
   handleKeyDown = e => {
     if (e.metaKey || e.ctrlKey || e.shiftKey || e.altKey) return
+
     switch (e.key) {
       case 'ArrowRight':
-      case ' ':
```

Comment:
Thanks for the PR! This is intentional – it advances with the space bar

Reply:
Oh cool. I thought that was for something, but not think in the space bar.